src/components/LoginPage.tsx

```
@@ -23,15 +23,38 @@ export function LoginPage({ onSkip }: LoginPageProps) {
-              className="btn btn-primary btn-lg gap-2"
-            >
-              <GitHub size={24} />
-              Login com GitHub
-            </button>
+          <div className="mb-6">
+            <h2 className="font-semibold text-lg mb-3 text-center">Escolha o nível de acesso:</h2>
+            <div className="flex flex-col gap-3">
+              <button
+                onClick={() => handleGitHubLogin('read')}
+                className="btn btn-outline btn-lg gap-2 flex-col h-auto py-4"
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)